CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contribution Guidelines
+
+Esri welcomes contributions to our open source projects on Github. Please see our [guidelines for contributing](https://github.com/esri/contributing).
+
+We do not currently accept new PRs for full features, for example a new Toolkit component or a significant change to an existing Toolkit component, but we do encourage you to submit issues for:
+
+* Bugs you find while using the Toolkit.
+* Requests for new Toolkit components or enhancements to existing Toolkit Components.
+
+## Contributing to the ArcGIS Maps SDK for Swift Toolkit
+
+This guide describes how you can contribute improvements to the ArcGIS Maps SDK for Swift Toolkit source code.
+
+1. Make sure you have a [GitHub account](https://github.com/signup/free).
+2. To add an enhancement or fix a problem, create an issue.
+    * Be descriptive: Clearly describe the issue including steps to reproduce and, if possible, a sample.
+    * If an enhancement, clearly describe the additional functionality and provide a good use case to support it.
+3. [Fork](https://help.github.com/articles/fork-a-repo) the [ArcGIS Maps SDK for Swift Toolkit repo](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/) on GitHub.
+4. Clone it to your local machine.
+5. Checkout the [main branch](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/tree/main).
+6. Create a new feature branch on your local machine. The name of the branch is not important.
+7. Write code to add an enhancement or fix a problem.  
+    * Follow the coding conventions already used in the Toolkit source code.
+    * Document your code.
+    * Make commits of logical units.  
+    * Use clear and descriptive commit messages.
+8. Self-review your code.
+9. Test your code.
+
+### Submitting changes
+
+1. Push the changes in your feature branch to your repo.
+2. Submit a [pull request](https://help.github.com/articles/using-pull-requests) against the "main" branch.
```

Comment:
`v.next` or `main` branch? I think this is one of the reasons we don't encourage users to make PRs to our repos - sometimes `v.next` contains code that depend on new APIs. I'm OK if we can correctly manage the user submitted changes on `main` - after all I've never seen a user submitted bug fix so far…